whois-update/src/main/java/net/ripe/db/whois/update/authentication/strategy/MntRefAuthentication.java

```diff
@@ -0,0 +1,35 @@
+package net.ripe.db.whois.update.authentication.strategy;
+
+import net.ripe.db.whois.common.dao.RpslObjectDao;
+import net.ripe.db.whois.common.rpsl.AttributeType;
+import net.ripe.db.whois.common.rpsl.ObjectType;
+import net.ripe.db.whois.common.rpsl.RpslObject;
+import net.ripe.db.whois.update.authentication.credential.AuthenticationModule;
+import net.ripe.db.whois.update.domain.PreparedUpdate;
+import net.ripe.db.whois.update.domain.UpdateContext;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+@Component
+public class MntRefAuthentication extends ReferenceAuthentication{
+
```

Comment:
Is this class authenticating for other attributes too apart from MNT_REF? If yes can you rename it? and why a change for other attributes needed?